Formula/go-boring.rb

```diff
@@ -0,0 +1,79 @@
+class GoBoring < Formula
+  desc "Go programming language with BoringCrypto"
+  homepage "https://go.googlesource.com/go/+/dev.boringcrypto/README.boringcrypto.md"
+  url "https://go-boringcrypto.storage.googleapis.com/go1.16.2b7.src.tar.gz"
+  sha256 "8991c28e30bdd9b34076c2871ab69aac7f306dba08d202aa7301f3adb54ad9f3"
```

Comment:
```suggestion
  sha256 "8991c28e30bdd9b34076c2871ab69aac7f306dba08d202aa7301f3adb54ad9f3"
  version "1.16.2b7"
```
This will allow you to include `b\d+` in the `livecheck` regex.